src/get-started/uninstall/index.md

```diff
@@ -0,0 +1,153 @@
+{% capture dart-files %}
+{{localappdata}}.dartServer
+{{appdata}}.dart
+{{appdata}}.dart-tool
+{% endcapture %}
+{% capture rm-dart-files %}
+{{prompt}} {{rm}} {{localappdata}}.dartServer,{{appdata}}.dart,{{appdata}}.dart-tool
+{% endcapture %}
+{% capture flutter-files %}{{appdata}}.flutter-devtools{% endcapture %}
+{% capture rm-flutter-files %}
+{{prompt}} {{rm}} {{flutter-files}}
+{% endcapture %}
+{% capture rm-pub-dir %}
+{{prompt}} {{rm}} {{localappdata}}Pub\Cache
+{% endcapture %}
+{% else -%}
+{% assign dirinstall='~/development' %}
+{% assign dirconfig='~/' %}
+{% assign path='~/development/' %}
+{% assign prompt='$' %}
+{% assign prompt-start=prompt %}
+{% assign rm = 'rm -rf ' %}
+{% assign rm-sdk = 'rm -rf ' | append: dirinstall | append: '/flutter' %}
+{% capture dart-files %}
+{{dirconfig}}.dart
+{{dirconfig}}.dart-tool
+{{dirconfig}}.dartServer
+{% endcapture %}
+{% capture rm-dart-files %}
+{{prompt}} {{rm}} {{dirconfig}}.dart*
+{% endcapture %}
+{% capture flutter-files %}
+{{dirconfig}}.flutter
+{{dirconfig}}.flutter-devtools
+{{dirconfig}}.flutter_settings
+{% endcapture %}
+{% capture rm-flutter-files %}
+{{prompt}} {{rm}} {{dirconfig}}.flutter*
+{% endcapture %}
+{% capture rm-pub-dir %}
+{{prompt}} {{rm}} {{dirconfig}}.pub-cache
+{% endcapture %}
+{% endcase -%}
+
+<div class="tab-pane {%- if id == 'windows' %} active {% endif %}" id="{{id}}" role="tabpanel" aria-labelledby="{{id}}-tab" markdown="1">
+
+## Uninstall the Flutter SDK
+{:.no_toc}
+
+This guide presumes that you installed Flutter in `{{path}}` on {{os}}.
+
+To uninstall the SDK, remove the `flutter` directory.
+
+```terminal
+{{prompt}} {{rm-sdk}}
+```
+
+## Remove configuration and package directories
+{:.no_toc}
+
+Flutter and Dart install additional directories in your home directory.
+These contain configuration files and package downloads.
+Each of the following procedures are _optional_.
+
+### Remove Flutter configuration files
+{:.no_toc}
+
+If you don't want to preserve your Flutter configuration,
+remove the following directories from your home directory.
+
+```terminal
+{{ flutter-files | strip }}
+```
+
+To remove these directories, run the following command.
+
+```terminal
+{{rm-flutter-files | strip}}
+```
+
+### Remove Dart configuration files
+{:.no_toc}
+
+If you don't want to preserve your Dart configuration,
+remove the following directories from your home directory.
+
+```terminal
+{{ dart-files | strip}}
+```
+
+To remove these directories, run the following command.
+
+```terminal
+{{rm-dart-files | strip}}
+```
+
+### Remove pub package files
+{:.no_toc}
+
+{{site.alert.important}}
+  If you want to remove Flutter but not Dart,
+  don't complete this section.
+{{site.alert.end}}
+
+If you don't want to preserve your pub packages,
+remove the `.pub-cache` directory from your home directory.
+
+```terminal
+{{rm-pub-dir | strip}}
+```
+
+</div>
+
+{% endfor -%}
+</div>
+{% comment %} End: Tab panes. {% endcomment -%}
```

Comment:
This section should live on dart.dev https://github.com/dart-lang/site-www/issues/3187

Reply:
**comment:** It can't just live there. The location of the configuration files is the same for both, but a complete removal should include all steps here.

Reply:
> It can't just live there. 
Yes it can and it should. We don't want duplicate docs between the sites.

Reply:
Good point, @johnpryan. We do want to avoid duplicatoin, so docs.flutter.dev should talk about removing Flutter. Then, if you want to remove Dart, go to dart.dev.

Reply:
This is not about deleting Dart per se. It's about deleting Flutter. If you delete Flutter, you delete Dart at the same time, not at some other time. It doesn't make sense to send the user elsewhere to delete a config directory for Dart which is part of their complete _Flutter_ install.